core/storage/suggestion/gae_models_test.py

```diff
@@ -0,0 +1,209 @@
+            self.target_id, self.target_version_at_submission,
+            suggestion_models.STATUS_IN_REVIEW, 'author_1',
+            'reviewer_1', 'reviewer_1', self.change_cmd, self.score_category,
+            'thread_1')
+        suggestion_models.SuggestionModel.create(
+            suggestion_models.SUGGESTION_EDIT_STATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            self.target_id, self.target_version_at_submission,
+            suggestion_models.STATUS_ACCEPTED, 'author_2',
+            'reviewer_2', 'reviewer_2', self.change_cmd, self.score_category,
+            'thread_2')
+        suggestion_models.SuggestionModel.create(
+            suggestion_models.SUGGESTION_EDIT_STATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            self.target_id, self.target_version_at_submission,
+            suggestion_models.STATUS_ACCEPTED, 'author_2',
+            'reviewer_3', 'reviewer_2', self.change_cmd, self.score_category,
+            'thread_3')
+        suggestion_models.SuggestionModel.create(
+            suggestion_models.SUGGESTION_EDIT_STATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            self.target_id, self.target_version_at_submission,
+            suggestion_models.STATUS_REJECTED, 'author_2',
+            'reviewer_2', 'reviewer_3', self.change_cmd, self.score_category,
+            'thread_4')
+        suggestion_models.SuggestionModel.create(
+            suggestion_models.SUGGESTION_EDIT_STATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            self.target_id, self.target_version_at_submission,
+            suggestion_models.STATUS_REJECTED, 'author_3',
+            'reviewer_3', 'reviewer_2', self.change_cmd, self.score_category,
+            'thread_5')
+
+    def test_target_type_choices_contains_delimiter(self):
+        for target_type in suggestion_models.TARGET_TYPE_CHOICES:
+            self.assertTrue(
+                suggestion_models.SUGGESTION_ID_DELIMITER not in target_type)
+
+    def test_score_type_contains_delimiter(self):
+        for score_type in suggestion_models.SCORE_TYPE_CHOICES:
+            self.assertTrue(
+                suggestion_models.SCORE_CATEGORY_DELIMITER not in score_type)
+
+    def test_create_new_object_succesfully(self):
+        suggestion_models.SuggestionModel.create(
+            suggestion_models.SUGGESTION_EDIT_STATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            self.target_id, self.target_version_at_submission,
+            suggestion_models.STATUS_IN_REVIEW, 'author_3',
+            'reviewer_3', 'reviewer_3', self.change_cmd, self.score_category,
+            'thread_6')
+
+        suggestion_id = suggestion_models.SuggestionModel.get_instance_id(
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            self.target_id, 'thread_6')
+
+        observed_suggestion = suggestion_models.SuggestionModel.get_by_id(
```

Comment:
observed_suggestion_model. Be careful to distinguish between model instances and domain objects by appending "_model" to the former.

Ditto elsewhere (if necessary).

Reply:
Done.